TESTING.md

```diff
@@ -14,6 +14,32 @@ Before you can run tests locally you must have:
 
         $ curl https://sdk.cloud.google.com | bash
 
+## System Dependencies
+
+Running the complete suite may require system dependencies. Most of these should be installable via typical package managers like `apt-get` and [brew](http://brew.sh/).
+
+For example, to install portaudio:
+
+    brew install portaudio
+
+If you have trouble installing a necessary system library, try filing an issue.
+
+### Troubleshooting
+
+#### PortAudio on OS X
+
+If you see the error
+    
+    fatal error: 'portaudio.h' file not found
+
+Try adding the following to your ~/.pydistutils.cfg file,
+substituting in your appropriate brew Cellar directory:
+
+    include_dirs=/usr/local/Cellar/portaudio/19.20140130/include/
+    library_dirs=/usr/local/YourUsername/homebrew/Cellar/portaudio/19.20140130/lib/
```

Comment:
I feel like it'd be more scalable to add this to the speech/README.md, and then reference that from here?